src/indexes.jl

```diff
@@ -0,0 +1,102 @@
+module IdxSet
+
+# WE USE MATPOWER'S DATA FORMAT
+
+function idx_bus()
+    BUS_I       = 1    # bus number (1 to 29997)
+    BUS_TYPE    = 2    # bus type (1 - PQ bus, 2 - PV bus, 3 - reference bus, 4 - isolated bus)
+    PD          = 3    # Pd, real power demand (MW)
+    QD          = 4    # Qd, reactive power demand (MVAr)
+    GS          = 5    # Gs, shunt conductance (MW at V = 1.0 p.u.)
+    BS          = 6    # Bs, shunt susceptance (MVAr at V = 1.0 p.u.)
+    BUS_AREA    = 7    # area number, 1-100
+    VM          = 8    # Vm, voltage magnitude (p.u.)
+    VA          = 9    # Va, voltage angle (degrees)
+    BASE_KV     = 10   # baseKV, base voltage (kV)
+    ZONE        = 11   # zone, loss zone (1-999)
+    VMAX        = 12   # maxVm, maximum voltage magnitude (p.u.)      (not in PTI format)
+    VMIN        = 13   # minVm, minimum voltage magnitude (p.u.)      (not in PTI format)
+
+    LAM_P       = 14   # Lagrange multiplier on real power mismatch (u/MW)
+    LAM_Q       = 15   # Lagrange multiplier on reactive power mismatch (u/MVAr)
+    MU_VMAX     = 16   # Kuhn-Tucker multiplier on upper voltage limit (u/p.u.)
+    MU_VMIN     = 17   # Kuhn-Tucker multiplier on lower voltage limit (u/p.u.)
+
+    return BUS_I, BUS_TYPE, PD, QD, GS, BS, BUS_AREA, VM, VA, BASE_KV, ZONE, VMAX, VMIN,
```

Comment:
This function returns a lot of indexes. How about returning a single named tuple, to ease the retrieval? See e.g. https://docs.julialang.org/en/v1/manual/types/#Named-Tuple-Types-1

Reply:
Yes, the named tuple makes sense. I just followed Matpower's trail and didnt really think this through.

Let me address this in a future PR. I want to focus on get to the RGM by next Tuesday.

Reply:
Let me also say. Using tables for the generator data and such also is not a good idea. I think I might rewrite it and just use a dictionary.

But, again, RGM first.